### PR TITLE
fix: Update DG formatting and site structure

### LINKED ISSUE #396: DG: Re-format
Reformat the DG after multiple merges.

- [x] Re-arrange implementations portion to have the same command flow as UG
- [x] Re-arrange manual testing to follow UG flow
- [x] Re-format manual testing sections to have consistent format
- [x] Ensure headers have consistent naming and hierachy
- [ ] Reformat images to proper size

### PR DESCRIPTION
**Pull request by delishad21**
Multiple DG updates 
- Format DG to have proper formatting
- Add HTML table of contents for the DG
- Remove the side nav bar for the DG
- Standardise Titles and implementation structure
- Standardise Appendix
- Add neater theme to markbind

View updated DG hosted [here](https://ba44-42-60-14-21.ngrok-free.app/DeveloperGuide.html)

Fixes #396

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1,15 +1,107 @@
...
+                    <li><a href="#test_importexam">Importing Exam Scores: `importExamScores`</a></li>
+                    <li><a href="#test_addscore">Adding a Persons's Exam Score: `addScore`</a></li>
+                    <li><a href="#test_editscore">Editing a Person's Exam Score: `editScore`</a></li>
+                    <li><a href="#test_deletescore">Deleting a Person's Exam Score: `deleteScore`</a></li>
+                    <li><a href="#test_statistics">Mean and Median of Exam Scores</a></li>
+                </ol>
+            <li><a href="#appendix_g">Appendix G: Effort</a></li>
+        </ol>
+    </li>
+
```
Comment (Zer0Legion) [inline docs/DeveloperGuide.md:100]: Great formatting with consistent naming of headers!

**Review by Zer0Legion — COMMENTED** [review]
These changes improve coherence of our DG as a whole

**Review by Pughal77 — COMMENTED** [review]
Nice! Now our DG has a better flow and structure

**Review by jayllo-c — COMMENTED** [review]
Might wanna ensure that the content page commands are properly in <code>COMMAND</code> and not \`COMMAND\`

**Review by jayllo-c — COMMENTED** [review]
Nice job adding all the center alignments as well 👍

**Review by jayllo-c — COMMENTED** [review]
Additional changes improve the formatting greatly, which add to the professionalism of the DG

**Pull request merged**